src/shared/components/case-viewer/case-viewer.component.spec.ts

```diff
@@ -132,7 +132,7 @@ const EVENTS: CaseViewEvent[] = [
 ];
 
 const METADATA: CaseField[] = [
-  {
+  Object.assign(new CaseField(), {
```

Comment:
would `<CaseField>` work here like we did above rather than doing `Object.assign`? Let's use one approach only if possible please

Reply:
No it didnt work in this case